libs/src/services/solanaWeb3Manager/transactionHandler.ts

```diff
@@ -96,25 +126,35 @@ class TransactionHandler {
       skipPreflight:
         skipPreflight === null ? this.skipPreflight : skipPreflight,
       feePayerOverride: feePayerOverride ? feePayerOverride.toString() : null,
-      retry
+      retry,
+      recentBlockhash: undefined as string | undefined
     }
 
     if (sendBlockhash || Array.isArray(signatures)) {
-      transactionData.recentBlockhash = (recentBlockhash || (await this.connection.getLatestBlockhash('confirmed')).blockhash)
+      transactionData.recentBlockhash =
+        recentBlockhash ??
+        (await this.connection.getLatestBlockhash('confirmed')).blockhash
     }
 
     try {
       const response = await this.identityService.solanaRelay(transactionData)
       return { res: response, error: null, errorCode: null }
-    } catch (e) {
-      const error =
-        (e.response && e.response.data && e.response.data.error) || e.message
+    } catch (e: any) {
+      const error = e.response?.data?.error || e.message
```

Comment:
Could type this guy

Reply:
kinda.. harder than it looks though. I have a slightly safer version coming following some guidance from:
https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript